test/testSyntax.js

```diff
@@ -1,4 +1,4 @@
-require('source-map-support').install()
```

Comment:
We should leave this in so that any errors come back with lines from the original source instead of the transpiled files.